### ISSUE TITLE
String.prototype.startsWith()

### ISSUE THREAD

**Issue by deukalion-io** [otacke/h5p-essay#35]
Hi: This is a really great plugin. We're using it on our Moodle site for training mental health service providers using case studies. Thank you for your work!

One issue I noticed in the console log when clicking the Check button on an Essay instance in IE 11 is that nothing happens. I think this is because (in part or perhaps totally) String.prototype.startsWith() isn't supported in IE11. 

I see that there are polyfills for this here: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Polyfill

I'm wondering if you'd consider adding that for backwards compatibility? Hopefully it's an easy fix. As much as I'd love to ignore IE11, a lot of the health care service sector where I'm from in Canada still uses it in their offices. 

Thanks again, 
Matt

**Comment by deukalion-io** [comment]
Thanks Oliver. Gotcha. I'm using 1.1.3. I think I remember manually uploading it to Moodle because as you said the H5P one was outdated.  

Just so I understand correctly: will the h5p-demo package you uploaded to the H5P tracker site eventually become the 1.1.4 with that IE11 bugfix, once it's reviewed?

**Comment by otacke** [comment]
Yes, the version number would be bumped for the next release. Otherwise, you couldn't update existing libraries (unless you switch your host platform to development mode).
